common/constants.go

```diff
@@ -83,6 +83,7 @@ const (
 	VisibilityAppName = "visibility"
 )
 
+// #nosec
```

Comment:
Can you provide what was the violation here? 
@wxing1292 to review once the violation is provided.

Reply:
Added a comment. This was flagged as potentially hardcoded credentials. Definitely a false positive.